src/gui/editors/PianoRoll.cpp

```diff
@@ -3918,31 +3918,33 @@ int PianoRoll::quantization() const
 
 void PianoRoll::quantizeNotes()
 {
-	NoteVector notes = getSelectedNotes();
+	if( hasValidPattern() ) {
```

Comment:
This is already better than before but I was also asking to use the following coding pattern:
```C++
if ( !hasValidPattern() )
{
  return;
}

NoteVector notes = getSelectedNotes();

// ...
// Everything that's inside the `if` statement follows here
```
This is an often used pattern when the scope of an `if` statement encompasses a whole method. By doing it like this you also get rid of one indentation level for the code that's inside the `if` clause which makes the code more readable.